explainer.md

```diff
@@ -229,9 +229,9 @@ Each `details` object contains:
 
 #### File Location
 
-To make use of the web-app-origin-association file, we suggest browsers locate it using a `<link rel="web-app-origin-association" href="/web-app-origin-association">` element in the header section of the main document at the origin's root path. Other formats such as `assetlinks.json` will have different requirements.
+To make use of the web-app-origin-association file, we suggest that association files be placed in relation to the root path of the origin. In order to match an origin with a `*.` prefix, we suggest that the corresponding association file be placed relative to the root path of the domain. Eg. an origin `*.contoso.com` could have a `web-app-origin-association` file at `contoso.com/.well-known/web-app-origin-association`.
```

Comment:
Do you want to specify "placed in a directory named ‘.well-known’ within the root path"?

Also, it’s not so much a suggestion as a requirement, right? It’s a predictable location that would enable the lookup code to avoid parsing the homepage HTML to discover the location of the file.

Reply:
Changed to "... placed in a `.well-known` directory within the root path of the origin..."

We worded the explainer originally to not prescribe how browsers should find WAOA files but we can revisit that when writing the web spec.